templates/native/react-redux/template.container.js

```diff
@@ -1,35 +1,35 @@
-import { bindActionCreators } from 'redux';
-import { connect } from 'react-redux';
-
-import TEMPLATE_UPPER_CAMEL_CASE_NAMEView from './TEMPLATE_KEBAB_CASE_NAME.view';
-import {
-    exampleAction,
-    exampleAsyncAction
-    /*
-    Import all the actions you wish to expose to the view here.
-    */
-} from './TEMPLATE_KEBAB_CASE_NAME.reducer';
```

Comment:
@YashKalaria-BTC I believe this was an error in the original code OR we change the approach to take from action JS?